### PR TITLE
OSS standardization changes

### PR DESCRIPTION
**Pull request by emmyoop**
resolves https://github.com/dbt-labs/dbt-core/issues/9221

Description
add CONTRIBUTING.md and dependabot alerts. Modify readme to bring this OSS repo up to the minimal standards for OSS at dbt Labs.

Also added templates for issues and PRs

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+4. Push changes to your fork
+5. Open a pull request against `dbt-labs/tree-sitter-jinja2` from your forked repository
+
+### dbt Labs contributors
+
+If you are a member of the `dbt-labs` GitHub organization, you will have push access to the `tree-sitter-jinja2` repo. Rather than forking `tree-sitter-jinja2` to make your changes, just clone the repository, check out a new branch, and push directly to that branch.
+
+## Setting up an environment
+
+TBD
```
Comment (emmyoop) [inline CONTRIBUTING.md:50]: Any tips here?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+
+## Running `tree-sitter-jinja2` in development
+
+### Installation
+
+TBD
```
Comment (emmyoop) [inline CONTRIBUTING.md:57]: How do we get installed?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+Once you're able to manually test that your code change is working as expected, it's important to run existing automated tests, as well as adding some new ones. These tests will ensure that:
+- Your code changes do not unexpectedly break other established functionality
+- Your code changes can handle all known edge cases
+- The functionality you're adding will _keep_ working in the future
+
+
+### Initial setup
+
+TBD
```
Comment (emmyoop) [inline CONTRIBUTING.md:73]: How do you get set up for testing?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+### Test commands
+
+TBD
```
Comment (emmyoop) [inline CONTRIBUTING.md:77]: Are there any test commands?  We usually put things like `make test` here.  If there's nothing we can delete the section.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+## Debugging
+
+TBD
```
Comment (emmyoop) [inline CONTRIBUTING.md:81]: How should the project be debugged?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+### Assorted development tips
+
+TBD
```
Comment (emmyoop) [inline CONTRIBUTING.md:85]: What are some general dev tips?

**Review by QMalcolm — APPROVED** [review]
Looks great! Thank you for doing this 🙂
